docs/differenceBetweenPBSGo-and-Java.md

```diff
@@ -27,6 +27,7 @@ and not the other for an interim period. This page tracks known differences that
 - PBS-Go fixed the IndexExchange-vs-IX issue. PBS-Java fix is in progress.
 - PBS-Java Rubicon adapter removes `req.cur` from request to XAPI.
 - PBS-Go use "60 seconds buffer + {bid,imp,mediaType}TTL" approach to determine caching TTL period.
+- PBS-Java has different names for system metrics. For example instead of `active_connections` it uses `vertx.http.servers.0.0.0.0:[PORT].open-netsockets.count`. See [Metrics](metrics.md) for details.
```

Comment:
Most likely `0.0.0.0` is also a variable as it stands for interface on which PBS is listening and it might be different